guides/development/hack_on_wanda.md

```diff
@@ -0,0 +1,49 @@
+# Hack on Wanda
+
+In order to run Wanda, the following is needed
+
+1. [Elixir](https://elixir-lang.org/), a programming language used to build scalable and maintainable applications.
+2. [Docker](https://docs.docker.com/get-docker/), a platform for developing, shipping, and running applications in containers.
+3. [Docker Compose](https://docs.docker.com/compose/install/), a tool for defining and running multi-container applications.
+
+## Development environment
+
+A `docker-compose` development environment is provided for easy setup and management of dependencies. To start the development environment, simply run the following command:
+
+```
+docker-compose up -d
+```
+
+This command starts a **Postgres** database and a **RabbitMq** instance, which is used for storage and communication respectively.
+
+## Setup Wanda
+
+After installing the dependencies, you need to perform some initial setup tasks before you can start Wanda. You can do this by running the following command:
```

Comment:
Maybe this needs revisiting as the dependencies are not installed yet at this point.
```suggestion
Before starting Wanda, some initial setup tasks are required. This can be achieved by running the following command:
```